frontend/components/Domain/Recipe/RecipeChips.vue

```diff
@@ -9,7 +9,7 @@
       color="accent"
       :small="small"
       dark
-      :to="isOwnGroup ? `${baseRecipeRoute}?${urlPrefix}=${category.id}` : undefined"
+      :to="`${baseRecipeRoute}?${urlPrefix}=${category.id}`"
```

Comment:
The chip stays clickable if the user is not allowed to access the RecipeExplorerPage which will then just redirect to login. 
This happens if the group is private and the recipe is accessed via a shared link by a non logged in user. 

So it is pretty niche occurance.